.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Report a reproducible bug or regression.
+title: ''
+labels: 'type: bug 🐞'
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        🐞 If you do need to report a bug, please follow the template below to increase the chances of your report being looked at.
+
+        🧱 If adequate, assign one of the `Module: XXX` labels to your issue when creating it.
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug description
+      description: A description of the encountered bug.
+    validations:
+      required: true
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce
+      description: A list of steps and/or code snippets required to reproduce the behavior. A live example (e.g. codesandbox, codepen) is preferred.
```

Comment:
A live example (e.g. VTK.js examples, codesandbox, codepen...)

Reply:
Yes!  Is there a template for one of the live example sites?  Based this one off of a @jourdain one a while back: https://stackblitz.com/edit/js-bqmqyf

Reply:
There is one listed in the performance template:
https://codepen.io/jourdain/pen/RQZWYa